.travis.yml

```diff
@@ -25,7 +17,8 @@ install:
     - python setup.py install
 
 script:
-    - coverage run -m tests.run_tests
+    # - coverage run -m tests.run_tests
+    pytest
 
 after_success:
     - coveralls
```

Comment:
I tried both and also liked code cover better; also, last time I checked, coveralls didn't do branch coverage